config/config.go

```diff
@@ -415,6 +415,15 @@ func (config ServerConfig) GetDbPort() int {
 	return config.DbPort
 }
 
+// GetDbPortOrDefault - Gets the database port from the given configuration, otherwise returns the default Postgres port (5432)
+func (config ServerConfig) GetDbPortOrDefault() int {
```

Comment:
Note for reviewer: The reason I made a separate function is that there is another caller of `GetDbPort` that is used for defining the self-managed system scope, which would be affected by changing the result from `0` to `5432`, so its best kept there: https://github.com/pganalyze/collector/blob/main/config/identify_system.go#L111